script_runner2.py

```diff
@@ -374,7 +374,7 @@ def _append_to_log_file(self, console_out):
             except Exception as ex:
                 trace = traceback.format_exc()
                 error = '{0}: {1}'.format(str(ex), trace)
-                self._append_to_error_buffer(error)
+                self.log.error(error)
```

Comment:
This was copied over from reqExec incorrectly. cexec does not have error buffers.